tensorboard/plugins/npmi/BUILD

```diff
@@ -43,6 +43,19 @@ py_test(
     ],
 )
 
+py_binary(
+    name = "npmi_demo",
+    srcs = ["npmi_demo.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":summary",
+        "//tensorboard:expect_tensorflow_installed",
+    ],
+    data = [
+        "mountains.txt"
```

Comment:
Maybe worth calling this demo_mountains.txt or something so that folks looking at the directory don't wonder whether this plays some critical role in the operation of the plugin :)

Reply:
Changed to python file, so this is not relevant anymore.